### ISSUE TITLE
usage in help for "cf stack" is incorrect

### ISSUE THREAD

**Issue by jbayer** [cloudfoundry/cli#426]
with 6.11.1 the output looks like this:

```
$ cf stack
FAILED
Incorrect Usage.

NAME:
   stack - Show information for a stack (a stack is a pre-built file system, including an operating system, that can run apps)

USAGE:
   cf stack

OPTIONS:
   --guid	Retrieve and display the given stack's guid. All other output for the stack is suppressed.
```

but it should be something like this where STACK_NAME is shown as a required user-provided input:

```
$ cf stack
FAILED
Incorrect Usage.

NAME:
   stack - Show information for a stack (a stack is a pre-built file system, including an operating system, that can run apps)

USAGE:
   cf stack STACK_NAME

OPTIONS:
   --guid	Retrieve and display the given stack's guid. All other output for the stack is suppressed.
```

**Comment by savitaashture** [comment]
Hi @jbayer 
This issue is fixed in the following PR please check..
https://www.pivotaltracker.com/n/projects/892938/stories/92694856

Thanks @savitaashture

**Comment by goehmen** [comment]
This is being fixed as part of PR #415
